rethinkdb/_export.py

```diff
@@ -259,11 +259,12 @@ def export_table(db, table, directory, options, error_queue, progress_info, sind
         with sindex_counter.get_lock():
             sindex_counter.value += len(table_info["indexes"])
         # -- start the writer
-        task_queue = SimpleQueue()
+        ctx = mp.get_context(mp.get_start_method())
```

Comment:
Multiprocessing has no get_context in python 2.7.x as far as I know. This would break the code for those users who are using export functionalities with python 2.7.x.